libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -527,20 +542,24 @@ impl<'a> JsiiImporter<'a> {
 		self.add_members_to_class_env(&jsii_class, is_resource, Phase::Preflight, &mut class_env, new_type);
 		if is_resource {
 			// Look for a client interface for this resource
-			let client_interface = jsii_class.docs.and_then(|docs| docs.custom).and_then(|custom| {
-				custom
-					.get("inflight")
-					.map(|fqn| {
-						// Some fully qualified package names include "@" characters,
-						// so they have to be escaped in the original docstring.
-						if fqn.starts_with("`") && fqn.ends_with("`") {
-							&fqn[1..fqn.len() - 1]
-						} else {
-							fqn
-						}
-					})
-					.and_then(|fqn| self.jsii_types.find_interface(fqn))
-			});
+			let client_interface = jsii_class
```

Comment:
Curious - what was the motivation to add `.add_ref()`s here?

Reply:
The reasoning is because rust yelled at me :)

The real reason (I think) is because the usage of closures causes the `docs` to get borrowed. This is normally fine, but above I do added another borrow so I needed to make sure we're only dealing with references here.